molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/index/MappingsBuilder.java

```diff
@@ -142,7 +142,7 @@ private static void createAttributeMappingContents(AttributeMetaData attr, boole
 				else
```

Comment:
What have you changed in molgenis-app/src/test/resources/emx_all_datatypes.xlsx?